pkgs/top-level/python-packages.nix

```diff
@@ -26314,6 +26314,25 @@ in {
     };
   };
 
+  vim-vint = buildPythonPackage rec {
```

Comment:
Okay. I thought it made sense, I also saw for example pylint being in python-packages.nix. But since the resulting binary isn't strictly related to python, I see why it should be a dedicated package. I can try that :+1: